Expensify/App
# Move font-face declarations to a separate CSS file

### Details
Moving the fonts that load in `index.html` to a separate css file so they can be used on other pages.

### Fixed Issues
No Issue.

### Tests
1. Load web and desktop
2. Build web
3. Verify the fonts load on all platforms

### Tested On


### Screenshots
❌
